masterlist.yaml

```diff
@@ -6477,6 +6479,26 @@ plugins:
       # version: 1.7
       - crc: 0x725F1E2E
         util: 'SSEEdit v3.2.1'
+  - name: 'Smilodon - Combat of Skyrim.esp'
```

Comment:
> Smilodon is a spin-off of Wildcat and its metadata are almost same as Wildcat.

In that case I'd place `Smilodon - Combat of Skyrim.esp` directly after the entry for `Wildcat - Combat of Skyrim.esp`, and before the `Wildcat - Realistic Damage Plugin.esp`.

The order of entries in the masterlist has no effect. 
It's perfectly fine to ignore the order. if a more appropriate placement is available.

A regex for shared metadata could then be added later without too much noise.
```yaml
  - name: '(Wildcat|Smilodon) - Combat of Skyrim.esp'
```
Smilodon also has a Realistic Damage Plugin, so I think they could also share metadata with a regex.

Reply:
I created a new entry `(Smilodon|Wildcat) - Combat of Skyrim\.esp` to share metadata. Please confirm if it looks all right. Thanks.